modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,36 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return object autoloaded data - autoload size, entries, option and its names.
+ */
+function perflab_aao_autoloaded_options() {
+	global $wpdb;
+	$autoload_summary                   = $wpdb->get_row( "SELECT SUM(LENGTH(option_value)) AS autoload_size,COUNT(*) AS autload_entries FROM `$wpdb->options` WHERE autoload='yes'" );
+	$autoload_summary->autoload_toplist = $wpdb->get_results( "SELECT option_name, LENGTH(option_value) AS option_value_length FROM `wp_options` WHERE autoload='yes' AND LENGTH(option_value) > 100 ORDER BY option_value_length DESC LIMIT 20" );
```

Comment:
I think it would be great if we could make the `100` a filterable value. You could introduce a `perflab_aao_autoloaded_option_table_threshold` filter. See https://github.com/WordPress/performance/pull/353/files#diff-ade5f4171eac2c17ebf91369994d78d2f65e57a3390f389069d4d6757a078f02R59-R66 for an example code that you could follow.